Unity-Technologies/com.unity.multiplayer.samples.bitesize
# Dgs/unity 6 update

### Description
- Updated Dedicated Game Server sample to Unity 6, and related packages to 1.1.0

### Issue Number(s)
- https://jira.unity3d.com/browse/MTT-8617

### Contribution checklist

### How to test
1. Clone this branch
2. Open multiplayer play mode
3. Set the current editor role as client or server
4. Enable 2 virtual players and set their role so that there are exactly 2 clients and 1 server (inlcuding the main editor)
5. Enter play mode
6. Check that no errors show up in the console
